### PR TITLE
feat: external functions

### PR DESCRIPTION
**Pull request by load1n9**

### CODE REVIEW

File: common/src/keyword.rs [hunk 1]
```diff
@@ -5,8 +5,9 @@ pub enum Keyword {
   Import,
   As,
   From,
-  Builtin,
   Export,
+  Extern,
+  Builtin,
```
Comment (eliassjogreen) [inline common/src/keyword.rs]: Is `builtin` ever used? And if so for what? Feels like it's not needed when we have extern right?

**Review by eliassjogreen — CHANGES_REQUESTED** [review]

**Comment by eliassjogreen** [comment]
Also fix fmt

File: common/src/keyword.rs (hunk 1 above)
Comment (load1n9) [reply common/src/keyword.rs]: true I'll fix that

**Comment by load1n9** [comment]
> Also fix fmt

fixed

**Review by eliassjogreen — APPROVED** [review]

**Pull request merged**